routine-listen-ap.h

```diff
@@ -1,13 +1,16 @@
 #ifndef __ROUTINE_LISTEN_AP__
 #define __ROUTINE_LISTEN_AP__
 
-bool sendWifiInfo(WiFiClient& client) {
+bool sendConfig(WiFiClient& client) {
   
   DynamicJsonBuffer jsonBuffer;
   JsonObject& json = jsonBuffer.createObject();
 
   json["ip"] = WiFi.localIP().toString();
```

Comment:
Why json keys for wifi are hardcoded, but for mqtt don`t?
What about using constants for wifi too?